capstone/src/main/resources/static/event.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="UTF-8">
+    <title> Create an Event </title>
+    <script src="script.js"> </script>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  
+  <body>
+
+    <div class="menu">
+      <nav>
+        <a href="map.html"> Campus Map </a>
+      </nav>
+    </div>    
+
+    <h1>Create an Event </h1>
+
+    <div class="event-form">
+      <form id="form" method="POST">
+
+        <label for="organizationID"> OrganizationID: </label>
+        <input type="number" id="organizationID" name="organizationID"> 
+        <br><br>
+
+        <label for="organizationName"> Organization Name: </label>
+        <input type="text" id="organizationName" name="organizationName">
+        <br><br>
+
+        <label for="university"> University: </label>
+        <input type="text" id="university" name="university">
+        <br><br>
+
+        <label for="eventTitle"> Event Title: </label>
+        <input type="text" id="eventTitle" name="eventTitle">
+        <br><br>
+
+        <label for="eventTime"> Event Time: </label>
+        <input type="time" id="eventTime" name="eventTime">
+        <br><br>
+
+        <label for="eventLocation"> Event Location: </label> <br>
+        <input type="number" id="eventLat" name="eventLat" placeholder="Event Latitude"> <br>
+        <input type="number" id="eventLng" name="eventLng" placeholder="Event Longitude">
```

Comment:
Is this input eventually going to be connected to a location search of some sort instead of latlong?

Reply:
Maybe they can click on the map to get the coordinates?
Alternatively, you can ask them to fill in an address, and use the [Geocoding API](https://developers.google.com/maps/documentation/javascript/geocoding).

As always, this is not something that needs to be done in this PR. I would file a separate issue for it.
For now, typing in the lat/long is fine. It's an iterative process.

Reply:
I'll look into geocoding, but the current setup of the form is just a lot of placeholder stuff